test/dummy/config/environments/test.rb

```diff
@@ -24,4 +24,10 @@
   I18n.enforce_available_locales = false
 
   config.active_support.test_order = :sorted
+  config.log_level = :debug
+  if Rails.version < '5'
+    # TODO: this needs to be addressed
```

Comment:
What needs to be addressed here?

Reply:
If I don't set this config in rails 4.2, it will raise a deprecation warning.  I'm not entirely sure there is more to do here or if that warning is just intended to warn people about about a change in behavior.

Reply:
I don't think a TODO here is appropriate, as there's nothing to fix. Instead, a comment explaining why this is needed for older Rails would be good.

Reply:
It was more of a reminder to myself that I forgot to remove.